### ISSUE TITLE
About reflective dataset

### ISSUE THREAD

**Issue by kaka45inablink** [ykdai/Flare7K#23]
Hi，Thanks for your great job first！
I wonder if BracketFlare is better than Flare7k-ReflectiveFlare dataset for reflective flare removal or not。

**Comment by ykdai** [comment]
Thank you for your question. 
In fact, they are for different situations. If you want to remove the smartphone's ghosting effect (bright floating spots which are quite common for Iphone), BracketFlare is specially designed for this sitation. If you want to remove general reflective flare for professional cameras, you may still need to use Flare7k-ReflectiveFlare.

**Comment by kaka45inablink** [comment]
Thanks for your reply.
I have three more questions :
1. Why are the two datasets for different situations ? 
2. Do images in Flare7k-ReflectiveFlare contain both reflective flare and corresponding light source ? 
3. Does Flare7k-ReflectiveFlare follow the Optical Center Symmetry Prior ?

**Comment by ykdai** [comment]
1. The patterns of flare are influenced by the light sources and lenses. When we proposed the Flare7K, we did not consider the effect of different light sources and did not notice the optical center symmetry prior. Thus, Flare7K cannot be used to detect and remove the ghosting effect accurately. That's why we have BracketFlare dataset.
2. Yes.
3. Since we have data augmentation for the flare's position, the Flare7k-ReflectiveFlare doesn't follow the Optical Center Symmetry Prior.
